ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py

```diff
@@ -1497,6 +1497,40 @@ def test_tensorflow_max_pool2d(
     )
 
 
+# max_pool3d
+@handle_frontend_test(
+    fn_tree="tensorflow.nn.max_pool3d",
+    data_format=df(data_format=st.sampled_from(["NDHWC"])),
```

Comment:
You can simply here use `data_format=st.sampled_from(["NDHWC", "NCDHW"])`

Reply:
Thanks @hirwa-nshuti, I have applied that for both test functions.